gui/scripts/syncthing/core/controllers/syncthingController.js

```diff
@@ -995,6 +995,9 @@ angular.module('syncthing.core')
             $scope.currentFolder.devices.forEach(function (n) {
                 $scope.currentFolder.selectedDevices[n.deviceID] = true;
             });
+            if (typeof $scope.currentFolder.order === 'undefined') {
+              $scope.currentFolder.order = "random";
+            }
```

Comment:
Is this even possible, given we're setting the value below?
Or is this paranoia?